ui/page/debug_page.go

```diff
@@ -128,3 +138,19 @@ func (pg *DebugPage) Layout(gtx C) D {
 	}
 	return components.UniformPadding(gtx, container)
 }
+
+func (pg *DebugPage) resetDexData() {
+	// Show confirm modal before resetting dex client data.
+	confirmModal := modal.NewInfoModal(pg.Load).
+		Title("Confirm DEX Client Reset").
+		Body("You may need to restart godcr before you can use the DEX again. Proceed?").
+		NegativeButton(values.String(values.StrCancel), func() {}).
+		PositiveButton("Reset DEX Client", func() {
+			if pg.Dexc().Reset() {
```

Comment:
```suggestion
			if pg.WL.MultiWallet.DexClient.Reset() {
```

This should follow same approach used here https://github.com/planetdecred/godcr/blob/989a13fead1759eae8a28f51f4c0b8e6b7eaf76c/ui/page/proposal/proposals_page.go#L121

Reply:
`Dexc().Reset()` does not return a fixed, stored value. It tries to delete the dexc database and do other stuff, then returns true if it succeeds. The return value is also only used here to determine if to show a success message or not, nothing more, don't see a reason to store the returned value in a struct property.